app/services/persistent-stateful-service.ts

```diff
@@ -25,10 +26,13 @@ export abstract class PersistentStatefulService<TState extends object> extends S
   init() {
     this.store.watch(
       () => {
-        return JSON.stringify(this.state);
+        return this.state;
```

Comment:
This change actually completely breaks the persistent stateful service, and it will no longer correctly persist your changes.  Vuex watchers are actually really poorly documented, but essentially this first function acts as a change detector.  It will run whenever the store changes, and will compare the return value to the last time it ran.  It does this with a simple `===` comparison.  Since `this.state` will be a reference to the same object, your bottom function will never get run.  This will result in changes to persistent stateful service state never actually getting persisted.

Reply:
Good to know, thanks